### ISSUE TITLE
Crash after paste text on instance running inside WSL2 with Windows Terminal

### ISSUE THREAD

**Issue by smonv** [zellij-org/zellij#1318]
**Basic information**

`zellij --version`: zellij 0.27.0
`tput lines`: 47
`tput cols`: 209
`uname -av` or `ver`(Windows): Linux DESKTOP-ETAP7D1 5.10.102.1-microsoft-standard-WSL2 #1 SMP Wed Mar 2 00:30:59 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux

**Further information**
Reproduction steps:

- Copy text from Microsoft Edge URL bar (example: DEV-4891)
- Paste text to current Zellij tab open inside WSL2 with Windows Terminal
- Zellij crash

**BACKTRACE**

```
❯ RUST_BACKTRACE=1 zellij


  × Thread 'stdin_handler' panicked.
  ├─▶ At /home/vtt/.cargo/registry/src/github.com-1ecc6299db9ec823/termwiz-0.15.0/src/readbuf.rs:47:25
  ╰─▶ range start index 18446744073709551613 out of range for slice of length 8
         0: <zellij_utils::errors::Panic as core::fmt::Display>::fmt
         1: miette::handlers::graphical::GraphicalReportHandler::render_causes
         2: zellij_utils::errors::fmt_report
         3: zellij_utils::errors::handle_panic
         4: std::panicking::rust_panic_with_hook
                   at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/panicking.rs:702:17
         5: std::panicking::begin_panic_handler::{{closure}}
                   at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/panicking.rs:588:13
         6: std::sys_common::backtrace::__rust_end_short_backtrace
                   at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/sys_common/backtrace.rs:138:18
         7: rust_begin_unwind
                   at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/panicking.rs:584:5
         8: core::panicking::panic_fmt
                   at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/core/src/panicking.rs:143:14
         9: core::slice::index::slice_start_index_len_fail
                   at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/core/src/slice/index.rs:35:5
        10: termwiz::readbuf::ReadBuffer::find_subsequence
        11: std::sys_common::backtrace::__rust_begin_short_backtrace
        12: core::ops::function::FnOnce::call_once{{vtable.shim}}
        13: <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
                   at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/alloc/src/boxed.rs:1853:9
            <alloc::boxed::Box<F,A> as core::ops::function::FnOnce<Args>>::call_once
                   at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/alloc/src/boxed.rs:1853:9
            std::sys::unix::thread::Thread::new::thread_start
                   at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/sys/unix/thread.rs:108:17
        14: start_thread
        15: __GI___clone3

  help: If you are seeing this message, it means that something went wrong.
        Please report this error to the github issue.
        (https://github.com/zellij-org/zellij/issues)

        Also, if you want to see the backtrace, you can set the `RUST_BACKTRACE` environment variable to `1`.
```

**Comment by imsnif** [comment]
This is fixed in main and will be fixed in 0.28.0. Will probably release it today.

**Comment by smonv** [comment]
> This is fixed in main and will be fixed in 0.28.0. Will probably release it today.

Awesome, thank for the fix and info.
